src/financial-assistance.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Financial Assistance
+---
+
+Our financial assistance programme exists to support attendees who might otherwise find it difficult to attend the conference.
+We have a budget set aside for this purpose.
+
+The deadline for applications is 17:00 (Europe/London) on Friday, 17th June 2022.
```

Comment:
Would suggest adding an are we meeting yet link to the time to make it easier for folks to track timezones